src/aspire/reconstruction/estimator.py

```diff
@@ -145,7 +147,7 @@ def apply_kernel(self, vol_coeff, kernel=None):
         vol = self.basis.evaluate(vol_coeff)
         # convolve_volume expects a 3-dimensional array
         # so we remove the first dimension of the volume, which is 1
-        vol = kernel.convolve_volume(np.squeeze(vol.asnumpy(), axis=0))
+        vol = Volume(kernel.convolve_volume(np.squeeze(vol.asnumpy(), axis=0)))
```

Comment:
I expect there is another way to do this.  Maybe vol[0] ?

Reply:
That's much cleaner. We still need to wrap in `Volume` since `convolve_volume()` does not return a `Volume`.  Do we want `convolve_volume` to return a Volume?

Reply:
I'd have to look if it is can of worms first. For now I think can skip that part. Thanks